ccloud/ccloud-stack/ccloud_stack_create.sh

```diff
@@ -37,7 +37,7 @@ ccloud::create_ccloud_stack $enable_ksqldb || exit 1
 
 echo
 echo "Validating..."
-SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
+SERVICE_ACCOUNT_ID=$(confluent kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
```

Comment:
Because of the breaking change for IDs of the service account

```suggestion
SERVICE_ACCOUNT_ID=$(confluent kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4 "-" $5;}')
```

Reply:
This took me a few minutes to understand 😅